periphery/tenderly/package.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
And why did you decide to move it to its own folder and package.json again?

Reply:
because I wasn't fully aware of how tenderly actions are built; I just assumed you were copypasting the js code to their interface, then I noticed it's a dev local process. This is now following the tenderly cli setup but doesn't litter the parent namespace with a tenderly dependency. I think in the first version I reviewed it was still mixed up a little. This version is now fully isolated and lives in the same context as dune to make it clear that this runs together.

Reply:
ah I see.
No it's exactly the way that I did it in my first commit except for the naming.